draggablepanel/src/main/java/com/github/pedrovgs/DraggableView.java

```diff
@@ -257,24 +279,48 @@ public boolean isClosed() {
    * @return true if the touch event is realized over the drag or second view.
    */
   @Override public boolean onTouchEvent(MotionEvent ev) {
+
     viewDragHelper.processTouchEvent(ev);
     if (isClosed()) {
       return false;
     }
     boolean isDragViewHit = isViewHit(dragView, (int) ev.getX(), (int) ev.getY());
     boolean isSecondViewHit = isViewHit(secondView, (int) ev.getX(), (int) ev.getY());
+    boolean interceptTap = viewDragHelper.isViewUnder(dragView, (int) ev.getX(), (int) ev.getY());
+
     if (isMaximized()) {
       dragView.dispatchTouchEvent(ev);
     } else {
       dragView.dispatchTouchEvent(cloneMotionEventWithAction(ev, MotionEvent.ACTION_CANCEL));
     }
+
+      switch(ev.getAction())
+      {
+          case MotionEvent.ACTION_DOWN:
+              x1 = ev.getX();
+              break;
+          case MotionEvent.ACTION_UP:
+              x2 = ev.getX();
```

Comment:
Please, don't use x1 and x2 as variable method. use `lastTouchActionDownXPosition` instead.

Review also `x2`, you can reduce the scope to method level instead of class level, can't you?